circle.yml

```diff
@@ -1,29 +1,16 @@
-sudo: false
-
-language: node_js
-
-node_js:
-  - "0.10.28"
-
-env:
-  global:
-    - NODE_ENV=travis
-
-services:
-  - rabbitmq
-
-addons:
-  hosts:
-    - rabbitmq
-
-before_install:
-  - npm install -g npm
-
-install:
+machine:
+ node:
+  version: 0.12.0
+ services:
+  - docker
+dependencies:
+ override:
   - npm install
-
-script:
-
+  - docker run -d --name rabbitmq rabbitmq:3.5-management
```

Comment:
@fakewaffle noticed you're not binding the ports to the host when starting this up,

You probably also need to add a `hosts` entry to have `rabbitmq` go to the right spot. https://circleci.com/docs/configuration#hosts